FreeRTOS/Lab-Project-FreeRTOS-FAT
# CI-CD-Updates

Description
-----------
Adding basic CI-CD, spelling, and formatting checks
Converts all line endings to LF if they were CRLF
Removes trailing whitespace from non-binary files
I recommend reviewing this with a whitespace free diff of the changes, which can be seen [here](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-FAT/pull/58/files)

Test Steps
-----------

CI-CD based build of the project

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
